fusor-ember-cli/app/controllers/undercloud-deploy.js

```diff
@@ -43,25 +48,21 @@ export default Ember.Controller.extend({
 
   actions: {
     deployUndercloud: function () {
-      var me = this;
```

Comment:
That's fine with me, I don't think I even changed that bit.

As far as I'm concerned you own the ember code, so make it look how you like. :)